static/js/StaticPages.jsx

```diff
@@ -2636,6 +2636,249 @@ const StaticHR = () =>
 const ConditionalLink = ({ link, children }) =>
   link ? <a href={link} target="_blank">{children}</a> : children;
 
+/*
+* Team Page
+*/
+
+
+
+const partition = (arr, prop) =>
```

Comment:
Its not clear what this function is doing. A line of documentation might be helpful. Also, can we move this into the component its being used with?